Resources/Prototypes/Traits/skills.yml

```
@@ -121,3 +121,25 @@
       inverted: true
       species:
         - Felinid
+
+- type: trait
+  id: Singer
+  category: Auditory
+  points: -3
+  requirements:
+    - !type:CharacterSpeciesRequirement
+      species:
+        - Vulpkanin # Naturally more vocal than humans
+        - Felinid
+  components:
+    - type: Singer
+      midiUi:
+        key: enum.InstrumentUiKey.Key
+        type: InstrumentBoundUserInterface
+    - type: Instrument
```

```suggestion
    - type: Instrument
```
I only looked at the C# sorry